front/app/containers/Admin/projects/project/analysis/InputPreview/components/CustomFieldLongFieldValues/BodyMultilocLongField.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { Text } from '@citizenlab/cl2-component-library';
+
+import { IInputsData } from 'api/analysis_inputs/types';
+import { IIdeaCustomField } from 'api/idea_custom_fields/types';
+
+import T from 'components/T';
+
+type Props = {
+  input: IInputsData;
+  customField: IIdeaCustomField;
+};
+
+const BodyMultilocLongField = ({ input, customField }: Props) => {
+  return (
```

Comment:
Most of the new files are these custom field specific LongField files, where I've just moved the definitions into their own files from the LongFieldValue file.